packages/grant-explorer/src/features/round/ViewRoundPage.tsx

```diff
@@ -757,3 +764,32 @@ const InactiveButton = (props: { label: string; testid: string }) => {
     </Button>
   );
 };
+
+const ReportCard = ({
+  chainId,
+  roundId,
+}: {
+  chainId: ChainId;
+  roundId: string;
+}) => {
+  const reportCardURL = `https://reportcards.gitcoin.co/${chainId}/${roundId}`;
+
+  return (
+    <a href={reportCardURL} target="_blank" className="group w-fit">
+      <div className="rounded-lg border border-grey-500 p-4 flex flex-col gap-2">
+        <div className="flex items-center gap-2">
+          <div className="rounded-lg bg-grey-100 group-hover:bg-grey-200 w-8 h-8 transition-all" />
+          <div>
+            <div className="bg-grey-100 w-9 sm:w-12 h-0.5 mb-1  group-hover:bg-grey-200 transition-all" />
+            <div className="bg-grey-100 w-9 sm:w-12 h-0.5  group-hover:bg-grey-200 transition-all" />
+          </div>
+        </div>
+        <span className="sm:text-base text-sm">
+          Check out
+          <br /> this round’s
+          <br /> report card!
```

Comment:
I would prefer not to use `<br />` for layout. Try setting a fixed width on the report card element instead and the text will wrap naturally.